introduce Code of Conduct (#365)
up-for-grabs/up-for-grabs.net: CODE_OF_CONDUCT.md

  professional setting

## Our Responsibilities

Project maintainers are responsible for clarifying the standards of acceptable
behavior and are expected to take appropriate and fair corrective action in
response to any instances of unacceptable behavior.

Project maintainers have the right and responsibility to remove, edit, or
reject comments, commits, code, wiki edits, issues, and other contributions
that are not aligned to this Code of Conduct, or to ban temporarily or
permanently any contributor for other behaviors that they deem inappropriate,
threatening, offensive, or harmful.

## Scope

This Code of Conduct applies both within project spaces and in public spaces
when an individual is representing the project or its community. Examples of
representing a project or community include using an official project e-mail
address, posting via an official social media account, or acting as an appointed
representative at an online or offline event. Representation of a project may be
further defined and clarified by project maintainers.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behavior may be

> reported by contacting the project team at **<conduct@brendanforster.com>**. All

Reviewer: Fiddling with DNS/Google to get `conduct@up-for-grabs.net` set up.

Reviewer 2: I believe up-for-grabs is hosted on Namecheap (at least, says `whois`). If so, here's a quick guide on free e-mail forwarding for domains there: https://www.namecheap.com/support/knowledgebase/article.aspx/308/2214/how-to-set-up-free-email-forwarding.  I think it would be possible to set up conduct@up-for-grabs.net to forward to one or more of your addresses without incurring a cost.

> reported by contacting the project team at **<conduct@brendanforster.com>**. All

Reviewer 3: This sounds like a good idea to me.

> reported by contacting the project team at **<conduct@brendanforster.com>**. All

Author: Yep, if we have a shared inbox for this I'm :thumbsup: with switching this over. LMK if you need anything from me.

> reported by contacting the project team at **<conduct@brendanforster.com>**. All

Reviewer: (On vacation this week, will fiddle as time allows.)

# Up-For-Grabs Code of Conduct

This document outlines how contributors to the Up-For-Grabs project
should interact with each other, and the process for resolving issues.

> Projects listed on Up-For-Grabs are not covered by this code of conduct.

Reviewer 2: I think this is worded nicely and clearly. 👍

Reviewer 4: Only because "should" is used in place of "must". In actual reality it codifies behaviour, and requires pledging allegiance to it to participate. How rosy it sounds is beside the point of how authoritarian that is.
Not that undermining meritocracy (which is what it is actually meant to do) isn't riding fundamentally on signalling virtue.
How you caught "deemed" and still believed in this I don't understand.

you should refer to the governance rules if you have enquiries about an issue

> with a specific project.

Reviewer: Rebound: 

> Projects listed on Up-For-Grabs are not covered by this code of conduct.
> Inquiries about a specific project should be addressed according to that project's
> governance rules.

## Our Pledge

In the interest of fostering an open and welcoming environment, we as
contributors and maintainers pledge to making participation in our project and

> our community a harassment-free experience for everyone, regardless of age, body

Reviewer 4: Is it very welcoming to single out people for the non-immutable characteristics?
Is there no universality to it, or do you for some reason need to do it for just these ones, is that the kind of community this is?

size, disability, ethnicity, gender identity and expression, level of experience,
nationality, personal appearance, race, religion, or sexual identity and
orientation.

## Our Standards

> Examples of behavior that contributes to creating a positive environment

Reviewer 2: ```suggestion
Behaviors that contribute to creating a positive environment
```

tiny nitpick suggestion on wording.

Reviewer 4: If you run the caveat on the one below, this doesn't appear to mean the same thing as what it replaces.
Not that it wasn't badly written in the first place, but alas.

> include:

Reviewer 2: ```suggestion
include, but are not limited to:
```

> * Using welcoming and inclusive language

Reviewer 4: How does prescribing a particular ideology of language contribute to a positive environment.
Where has compelled speech ever done anything of the sort?

* Being respectful of differing viewpoints and experiences
* Gracefully accepting constructive criticism
* Focusing on what is best for the community
* Showing empathy towards other community members

> Examples of unacceptable behavior by participants include:

Reviewer 2: ```suggestion
Unacceptable behavior by participants includes, but is not limited to:
```

Reviewer 4: This is already superfluous to how project maintainers are meant to react to "any unacceptable behaviour" below.
However, that is only an expectancy.
"Unacceptable" thus functionally maintains parity with "acceptable".
The only "responsibility" is to instrument censorship for anything that isn't aligned to the CoC, or for "behaviors that they deem inappropriate, threatening, offensive, or harmful."
That is a police state with the law above the people, and its enactor. Made worse in the newer version btw.
In it the least amount of vested interest is tied to the same vagueness and whim, with zero oversight.

behavior and are expected to take appropriate and fair corrective action in
response to any instances of unacceptable behavior.

Project maintainers have the right and responsibility to remove, edit, or
reject comments, commits, code, wiki edits, issues, and other contributions
that are not aligned to this Code of Conduct, or to ban temporarily or
permanently any contributor for other behaviors that they deem inappropriate,
threatening, offensive, or harmful.

## Scope

This Code of Conduct applies both within project spaces and in public spaces
when an individual is representing the project or its community. Examples of
representing a project or community include using an official project e-mail
address, posting via an official social media account, or acting as an appointed
representative at an online or offline event. Representation of a project may be
further defined and clarified by project maintainers.

## Enforcement

Instances of abusive, harassing, or otherwise unacceptable behavior may be
reported by contacting the project team at **<conduct@brendanforster.com>**. All
complaints will be reviewed and investigated and will result in a response that

> is deemed necessary and appropriate to the circumstances. The project team is

Reviewer 2: ```suggestion
is necessary and appropriate to the circumstances. The project team is
```

`deemed` is an interesting word choice here because there's no implicit understanding of who would "deem" it so. 

Recommend removing the word, because it phrases it from the stance of the authors of the content (in this case, the maintainers), which seems appropriate.

Reviewer 4: Yes there is, "`Project maintainers`". Changed to "`Community leaders`" in Covenant CoC 2.0.

```

Project maintainers are responsible for clarifying the standards of acceptable
behavior and are expected to take appropriate and fair corrective action in
response to any instances of unacceptable behavior.

Project maintainers have the right and responsibility to remove, edit, or
reject comments, commits, code, wiki edits, issues, and other contributions
that are not aligned to this Code of Conduct, or to ban temporarily or
permanently any contributor for other behaviors that they deem inappropriate,
threatening, offensive, or harmful.
```

An anti-meritocratic and complex text-file, is supposed to undo antisocial behavior how?
By making sure unwritten rules worthy of mention are somehow less poignant than disobeying a text file for said people.

Sure the updated one goes even further in attracting anti-social people and granting them ultimate power, but the point remains escalation of conflict beyond the control of individuals involved.

`Other conduct which could reasonably be considered inappropriate in a professional setting`
To think this is a good idea I find puzzling.

Just as much as "harassment-free" is unattainable, it is also redundant to law. If the focus isn't intent, but here and other places equated to the ability and amount of offense taken, how do you communicate anything?
Undoing sexuality is going to work how?

The control group here is the 5 years without one.
